spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -288,182 +313,121 @@ public interface PolledBarista {
-    }
+public void sayHello(String name) {
+    source.output().send(MessageBuilder.withPayload(name).build());
 }
 ----
 
-[[spring-cloud-stream-overview-injecting-channels-directly]]
-===== Injecting Channels Directly
-
-Bound channels can be also injected directly, as shown in the following example:
+_Autowire individual channel_
 
 [source, java]
 ----
-@Component
-public class SendingBean {
-
-    private MessageChannel output;
-
-    @Autowired
-    public SendingBean(MessageChannel output) {
-        this.output = output;
-    }
-
-    public void sayHello(String name) {
-         output.send(MessageBuilder.withPayload(name).build());
-    }
-}
-----
+@Autowire
+private MessageChannel output;
 
-If the name of the channel is customized on the declaring annotation, that name should be used instead of the method name.
-Consider the following declaration:
-
-[source,java]
-----
-public interface CustomSource {
-    ...
-    @Output("customOutput")
-    MessageChannel output();
+public void sayHello(String name) {
+    output.send(MessageBuilder.withPayload(name).build());
 }
 ----
 
-Given that declaration, the channel is injected as shown in the following example:
+You can also use standard Spring's `@Qualifier` annotation for cases when channel names are customized or in a multiple channesl scenarios specifically named channel is required
 
 [source, java]
 ----
-@Component
-public class SendingBean {
-
-    private MessageChannel output;
-
-    @Autowired
-    public SendingBean(@Qualifier("customOutput") MessageChannel output) {
-        this.output = output;
-    }
-
-    public void sayHello(String name) {
-         this.output.send(MessageBuilder.withPayload(name).build());
-    }
-}
+@Autowire
+@Qualifier("myChannel")
+private MessageChannel output;
 ----
 
 [[spring-cloud-stream-overview-producing-consuming-messages]]
-==== Producing and Consuming Messages
+=== Producing and Consuming Messages
 
 You can write a Spring Cloud Stream application by using either Spring Integration annotations or Spring Cloud Stream's `@StreamListener` annotation.
-The `@StreamListener` annotation is modeled after other Spring Messaging annotations (such as `@MessageMapping`, `@JmsListener`, `@RabbitListener`, and so on) but adds content type management and type coercion features.
+The `@StreamListener` annotation is modeled after other Spring Messaging annotations (such as `@MessageMapping`, `@JmsListener`, `@RabbitListener`, and so on) but
+conviniences such as content based routing and others.
+
+==== Native Spring Integration Support
 
-===== Native Spring Integration Support
+Spring Cloud Stream is build on the concepts and patterns defined by http://www.enterpriseintegrationpatterns.com/[Enterprise Integration Patterns] and relies
+in its internal implementation on an already established and popular implementation of Enterprise Integration Patterns within Spring portfolio of projects -
```

Comment:
Replace "within Spring portfolio of projects -" with "within the Spring portfolio of projects:"